otter/constants.py

```diff
@@ -36,6 +37,10 @@ def get_service_configs(config):
             'name': config['cloudLoadBalancers'],
             'region': config['region'],
         },
+        ServiceType.CLOUD_ORCHESTRATION: {
```

Comment:
Can this service be optional just like metrics and cloudfeeds is? It will be nice to run otter without mandating this service.